combat-trainer.lic

```diff
@@ -213,6 +213,9 @@ class LootProcess
     @ritual_type = thanatology['ritual_type'].downcase
     echo("  @ritual_type: #{@ritual_type}") if $debug_mode_ct
 
+    @cycle_rituals = false
```

Comment:
Well, I ran through some changes and I need to keep a second variable.  @Ritual_type overrites itself to the preferred ritual so cycle is no longer available to test. It would lose the cycle function at that point.  I can either set cycle at the beginning or do it within cycle rituals itself.   Setting the variable at the beginning seems to be the cleanest way.